### DISCUSSION TITLE
Create an online Prisma query tester?

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#23246] [Q&A] [answered]
### Question

It would be great to have an online environment where to test queries and their return types. Would you consider implelementing this idea? I think It wouldn't be that difficult or if something similar already exist, can you please provide a link? Thanks!
Why not extend Prisma Editor with such feature ( https://prisma-editor.vercel.app/ )
### How to reproduce (optional)

Nothing to reproduce. Just an example of what I mean:
https://sqltest.net/

**Comment by nurul3101** [comment, marked as answer]
Hey @vtonchev 👋 

Do you mean something like https://playground.prisma.io/?
Have you tried it out?

**Reply by ghost** [reply]
yes, exactly. Thank you. I don't know why it's imposible to find it in google

**Reply by samudotlol** [reply]
Seems like this is under construction now, awh man

**Reply by valentinoPereira** [reply]
@nurul3101 Hey buddy, when will it be available? showing under construction.
